Configuration/StandardSequences/python/Eras.py

```diff
@@ -60,7 +60,7 @@ def __init__(self):
         # Scenarios further afield.
         # Phase2 is everything for the 2023 (2026?) detector that works so far in this release.
         # include phase 1 stuff until phase 2 tracking is fully defined....
-        self.Phase2 = cms.ModifierChain( self.phase1Pixel, self.trackingPhase1, self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
+        self.Phase2 = cms.ModifierChain( self.phase2_common, self.phase2_tracker, self.phase2_hgcal, self.phase2_muon )
```

Comment:
I believe phase2TkFlat and phase2TkTilted include all the customizations necessary for each version of the Phase2 tracker. The Phase1 customizations should not be necessary. @boudoul, can you confirm?